packages/edgeworth/src/components/Content.tsx

```diff
@@ -153,7 +154,13 @@ export class Content extends React.Component<ContentProps, ContentState> {
             );
           }
         }
-        const synth = new Synthesizer(env, setting, subResult, seed);
+        const synth = new Synthesizer(
```

Comment:
`Synthesizer` is really supposed to take in a `DomainEnv` and an optional `SubstanceEnv`. Since we have additional PRs depending on this one. I'd prefer to do this myself in another PR after this one is merged.